Automattic/wp-calypso
# Me: Add setting to disable gravatar

#### Changes proposed in this Pull Request

* This is a continuation of #31624, in simplified form. On the profile settings we now offer a toggle for a user to disable their gravatar profile and associated images from appearing:

<img width="663" alt="Screen Shot 2021-04-09 at 12 34 58 PM" src="https://user-images.githubusercontent.com/789137/114225804-01a4f580-9930-11eb-92a0-90333bba943e.png">

#### Testing instructions

**This requires changes to the wp.com api and gravatar to work**, see D59876-code and D59883-code for further instructions. Once that is set up, you can:

 * Head over to `/me`
 * Enable the hidden profile toggle and click save.
 * Head over to gravatar.com, your profile and avatar images should be disabled.
 * Turn the toggle off again.
 * Refresh gravatar, and your profile should be enabled.

@catscheer I added you here so you can have a look at the copy and general functionality!
